components/inputs/README.md

```diff
@@ -111,6 +112,30 @@ search.addEventListener('d2l-input-search-searched', (e) => {
 
 When the input is cleared, the same event will be fired with an empty value.
 
+## Select Lists
+
+Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class.
+
+Note: in order for RTL to function correctly, make sure your component uses the `RtlMixin`.
```

Comment:
This kind of sucks, but is unavoidable currently. We're investigating using CSS logical properties so that we can write RTL-agnostic CSS, but it's not supported in old-Edge currently.